tools/doc/mesh-part.1.scd

```diff
@@ -117,10 +117,10 @@ These algorithms improve partitions by only taking weights into account.
 Geometric partitioning algorithms:++
 These algorithms create partitions using cell coordinates.
 
-	*rcb*,PART_COUNT,[TOLERANCE=0.05]
+	*rcb*,RECURSIVE_BIPART_COUNT,[TOLERANCE=0.05]
```

Comment:
What is a "RECURSIVE_BIPART_COUNT" ? Is it the number of recursions (log2(p/2)), the number of bipartitions (p/2) or the targeted number of parts (p) ?

Reply:
Ah right, this is indeed an oversight!  In the code this parameter is spelled `iter_count` for iteration count, though I'm all in for something clearer.

Should you change the name here please do the same in the Rust API:

https://github.com/LIHPC-Computational-Geometry/coupe/blob/master/src/algorithms/recursive_bisection.rs#L825

https://github.com/LIHPC-Computational-Geometry/coupe/blob/master/src/algorithms/recursive_bisection.rs#L951

Reply:
Before changes `PART_COUNT` referred to the number of times a recursive bi-partitioning was performed, e.g. with value 1 (rep 2) it leads to 2 (resp 4) parts. This naming being kind of misleading when compared to other algorithms descriptions, I proposed to change it.
Objections on this rename ?